### PR TITLE
Initial commit of VPC FlowLogs Analysis

### PR DESCRIPTION
**Pull request by lgandras**

### CODE REVIEW

**Comment by jaketf** [comment]
@rosmo can you please review this?

File: tools/vpc-flowlogs-analysis/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+vpc_project_ids                = ["bigorg-hub-prod-prj-e126d530"]
```
Comment (jaketf) [inline tools/vpc-flowlogs-analysis/terraform.tfvars:2]: can you uncommit this file or make it terraform.tfvars.example and replace these job ids with place holders?

File: tools/vpc-flowlogs-analysis/variables.tf [hunk 2]
```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2020 Google LLC
```
Comment (jaketf) [inline tools/vpc-flowlogs-analysis/variables.tf:2]: double check date in all license headers.
```suggestion
 * Copyright 2021 Google LLC
```

File: tools/vpc-flowlogs-analysis/variables.tf [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+variable "location" {
+  type        = string
+  description = "GCP location, i.e. (multi-)region, where resources will be created. The list of all available values can be found under https://cloud.google.com/storage/docs/locations#available_locations."
+  default     = "EU"
+}
+
+variable "logs_project_id" {
+  type        = string
+  description = "Id of the project where the BigQuery dataset with the VPC flowlogs will be stored."
```
Comment (jaketf) [inline tools/vpc-flowlogs-analysis/variables.tf:48]:
```suggestion
  description = "ID of the project where the BigQuery dataset with the VPC flowlogs will be stored."
```

File: README.md [hunk 4]
```diff
@@ -105,6 +105,7 @@ creates relationships between assets and outputs a format compatible with [graph
 * [SLO Generator](tools/slo-generator/) - A Python package that automates computation of Service Level Objectives, Error Budgets and Burn Rates on GCP,  and export the computation results to available exporters (e.g: PubSub, BigQuery, Stackdriver Monitoring), using policies written in JSON format.
 * [Snowflake_to_BQ](tools/snowflake2bq/) - A shell script to transfer tables (schema & data) from Snowflake to BigQuery.
 * [STS Job Manager](tools/sts-job-manager/) - A petabyte-scale bucket migration tool utilizing [Storage Transfer Service](https://cloud.google.com/storage-transfer-service)
+* [VPC FlowLogs Analysis](tools/vpc-flowlogs-analysis) - A configurable Sink + BigQuery report that shows traffic grouped by target IP address ranges.
```
Comment (rosmo) [inline README.md:108]: Nitpick, but I'd love to be consistent about "VPC Flow Logs" spelling :) Otherwise, LGTM!

**Review by jaketf — APPROVED** [review]
LGTM

**Pull request merged**